src/entrypoints/index.ejs

```diff
@@ -64,6 +64,7 @@
     <title><%= options.title %></title>
   </head>
   <body>
+    <div id="__mausapp"></div>
```

Comment:
Awesome! I thought about doing this while I changed the `MobileScreenFallback`, but I thought that the PR was already big enough. Thanks!